src/components/leaderboard/view/leaderboardView.js

```diff
@@ -47,8 +47,8 @@ class LeaderboardView extends PureComponent {
           options={VALUE.map(val => intl.formatMessage({ id: `leaderboard.${val}` }))}
           selectedOptionIndex={selectedIndex}
           defaultText={intl.formatMessage({ id: `leaderboard.${VALUE[0]}` })}
-          onDropdownSelect={selectedIndex =>
-            fetchLeaderBoard(FILTER_OPTIONS[selectedIndex], selectedIndex)
+          onDropdownSelect={selectedIndexM =>
```

Comment:
what is M meaning?

Reply:
It is just making variable different than already used variable name